### PR TITLE
 testing/py-tinycss: upgrade to 0.4

### PR DESCRIPTION
**Pull request by stwa**
Depends on #5678

### CODE REVIEW

**Comment by andypost** [comment]
there's a commits from other updates, please split them

File: testing/py-cairocffi/APKBUILD [hunk 1]
```diff
@@ -1,40 +1,54 @@
...
-	local i
-	cd "$_builddir"
-	for i in $source; do
-		case $i in
-		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
-		esac
-	done
-}
+depends="py-cffi cairo gdk-pixbuf"
+options="!check"
```
Comment (andypost) [inline testing/py-cairocffi/APKBUILD]: it needs comment why check disabled when `check()` exists

**Comment by andypost** [comment]
@stwa please remove commit for configparser, it already added to testing

**Comment by stwa** [comment]
@andypost removed (thought Github removes it when reapplying PR on master)

**Pull request closed without merging**